<launch>

  <arg name="panels" default="false" />

  <param name="default_trajectory_folder"  value="$(find brl_drones)/launch" />
  <node name="traj_gui" pkg="brl_drones" type="csv_trajectory_gui.py" >
    <remap from="cmd_traj" to="bundled_traj" />
  </node>

  <node name="trajectory_publisher" pkg="brl_drones" type="trajectory_publisher.py" >
      <rosparam param="sink_topics">['drone1/cmd_traj', 'drone2/cmd_traj', 'drone3/cmd_traj']</rosparam>
      <param name="source_topic" value="bundled_traj" />
  </node>

  <node name="robot_state_publisher" pkg="robot_state_publisher" type="state_publisher" />

  <node name="rviz" pkg="rviz" type="rviz" args="-d $(find brl_drones)/urdf/drones.rviz"/>

  <param name="use_gui" value="false"/>
  <node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher">
    <rosparam param="source_list">['drone1/joint_states', 'drone2/joint_states', 'drone3/joint_states']</rosparam>
  </node>


<param name="robot_description" command="$(find xacro)/xacro.py /Users/rob/Desktop/fndws/src/brl_drones/urdf/myurdf_3.urdf.xacro" />

  <include file="$(find brl_drones)/launch/single_virtual.launch">
    <arg name="drone_name" value="drone1" />
    <arg name="use_panel" value="$(arg panels)" />
  </include>

    <include file="$(find brl_drones)/launch/single_virtual.launch">
    <arg name="drone_name" value="drone2" />
    <arg name="use_panel" value="$(arg panels)" />
  </include>

    <include file="$(find brl_drones)/launch/single_virtual.launch">
    <arg name="drone_name" value="drone3" />
    <arg name="use_panel" value="$(arg panels)" />
  </include>

  </launch>
